src/tini.c

```diff
@@ -219,6 +226,16 @@ int spawn(const signal_configuration_t* const sigconf_ptr, char* const argv[], i
 			return 1;
 		}
 	}
+
+	// Should unshare happen here, or in do_execvp / the child?
+	if (getenv(TINI_UNSHARE)) {
+		if (unshare(CLONE_NEWCGROUP)) {
+			PRINT_FATAL("Unable to unshare new cgroup namespace: %s", strerror(errno));
+			return 1;
+		}
```

Comment:
Useful to log anything that we attempted to unshare? Otherwise may be hard to tell/debug if attempted to share or not.